internal/evidence/pool.go

```diff
@@ -22,6 +22,16 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
+// key prefixes
+// NB: Before modifying these, cross-check them with those in
+// * internal/store/store.go    [0..4, 13]
+// * internal/state/store.go    [5..8, 14]
+// * internal/evidence/pool.go  [9..10]
+// * light/store/db/db.go       [11..12]
+// TODO(sergio): Move all these to their own package.
+// TODO: what about these (they already collide):
+// * scripts/scmigrate/migrate.go [3]
```

Comment:
this is part of the key migration script—does it collide, or is it using the "correct" prefix in an idiosyncratic way? (It should be the latter I hope, but possibly I introduced a bug)

Reply:
As far as I could see it refers to "seenCommit", and is consistent with the definition in the block store. I added a comment in this sense [here](https://github.com/tendermint/tendermint/pull/8673/commits/85caf47a84703be866d3ffeffb68b8709ae14640#diff-b26d414d0eb20deebca0d2b785bd3e1090aa1a7a182e7dc30640d68c53cb40daR659)